lib/component.js

```diff
@@ -213,6 +219,7 @@ class Component extends WebComponent {
     // appState and isStateShared of child components will be overwritten by parent/root
     // when the component is connected to the hierarchy
     this.state = {};
+    this.attrs= {};
```

Comment:
I thought we had eslint hooked up here? Surprised it hasn't complained.

Reply:
We still eslint everything. We've never enabled the `space-infix-ops` rule because AFAIK it's been an age old debate since the time I joined. 

We wanted to keep things consistent with python in function calls e.g `function(a=true, b='hello) {`. eslint doesn't give us a rule per operator so we didn't enabled it at the sacrifice of little things like this slipping in.